datastore/db/queries/machine_learning/__init__.py

```diff
@@ -404,8 +406,10 @@ def get_model_id_from_name(cursor,model_name:str):
         )
         model_id=cursor.fetchone()[0]
         return model_id
+    except(ValueError):
+        raise NonExistingTaskEWarning(f"Warning: the given model '{model_name}' was not found")
     except(Exception):
-        raise PipelineNotFoundError(f"Error: model not found for model name : {model_name}")
+        raise PipelineCreationError("Error: model not found")
```

Comment:
```suggestion
        raise PipelineNotFoundError(f"Error: model not found for model name : {model_name}")
```
The PipelineNotFoundError should be raised here